tunnel/draft-ietf-perc-dtls-tunnel.md

```diff
@@ -379,16 +349,47 @@ additional indicators to determine when an endpoint has disconnected.
 
 ## Key Distributor Tunneling Procedures
 
+Each TLS tunnel established between the media distributor and the
+key distributor **MUST** be mutually authenticated.
+
 When the media distributor relays a DTLS message from an endpoint, the
 media distributor will include an association identifier that is
 unique per endpoint-originated DTLS association.  The association
 identifier remains constant for the life of the DTLS association.  The
 key distributor identifies each distinct endpoint-originated DTLS
 association by the association identifier.
 
+When processing an incoming endpoint association, the key distributor
+**MUST** extract the `tls_id` value transmitted in the `ClientHello`
+message and match that against `tls-id` value the endpoint transmitted
+via SDP.  If the values in SDP and the `ClientHello` do not match, the
```

Comment:
Do we want to add wording here to indicate that this might be more something like a lookup table?
Something like "If the values in the `ClientHello` does not match any `tls-id` values from still pending client SDP's..."